internal/api/attestationconfigapi/cli/main.go

```diff
@@ -148,8 +152,32 @@ func runCmd(cmd *cobra.Command, _ []string) (retErr error) {
 	return nil
 }
 
+func allEqual[T comparable](args ...T) bool {
```

Comment:
Please add a unittest with at least the following cases:
1. one input arg
2. three input args that are equal
3. three input args where second and third element are different

Args should be snpReports.